org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -623,6 +672,33 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
             }
         }
 
+        /// <summary>
+        /// Set the enabled state of the Front plate Raw Image if any.
+        /// </summary>
+        /// <param name="enable">True to enable, false to disable</param>
+        protected void SetEnableFrontPlateRawImage(bool enable)
+        {
+            GameObject frontPlateGameObject = transform.Find(FRONT_PLATE_NAME).gameObject;
+            UnityEngine.UI.RawImage frontPlateRawImage = frontPlateGameObject?.GetComponent<UnityEngine.UI.RawImage>();
+            if (frontPlateRawImage != null)
+            {
+                frontPlateRawImage.enabled = enable;
+            }
+        }
+
+        /// <summary>
+        /// Set the enabled state of the CanvasElementRoundedRectMonoBehaviour MonoBehaviour if any.
+        /// </summary>
+        /// <param name="enable">True to enable, false to disable</param>
+        protected void SetEnabledCanvasElementRoundedRectIfAny(bool enable)
+        {
+            Microsoft.MixedReality.GraphicsTools.CanvasElementRoundedRect canvasElementRoundedRect = gameObject.FindAncestorComponent<Microsoft.MixedReality.GraphicsTools.CanvasElementRoundedRect>();
```

Comment:
This feels like it could be cached as well.

Reply:
It isn't odd that a child element would be directly modifying the enabled state of some parent component higher up in the hierarchy based on the child element's hover events?  At least built-in like this.  If that is needed in a particular use case, couldn't a dev instead hookup such things with events in the Inspector?

Reply:
Cached in latest commit, thank you for the feedback.